visual-regression/package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "start": "web-dev-server --open demo/ --node-resolve",
     "start:watch": "web-dev-server --open demo/ --node-resolve --watch",
+    "pretest": "web-test-runner test/**/*.test.js --coverage --node-resolve --update-visual-baseline",
```

Comment:
This previously failed because there were no baseline screenshots. We should probably make a pass at how you could cache screenshots from Actions, but I don't use them in my own projects, so I don't know anything about them... this builds a cache and then tests it, which is similar for this context.